shared/src/common/enums/user/user-validation-message.enum.ts

```diff
@@ -1,6 +1,8 @@
 const UserValidationMessage = {
   EMAIL_REQUIRE: 'Email is required',
   EMAIL_WRONG: 'Email is wrong',
+  EMAIL_MIN_LENGTH: 'Email length must be at least {#limit} characters long',
```

Comment:
use enum value instead

Reply:
`UserValidationRule` is received as `undefined` inside of this file, not sure why. Although the values are received perfectly fine inside of validation-schema file. Also, VSCode shows the values correctly, when hovering over it. 
![image](https://user-images.githubusercontent.com/61088675/183395615-74601e4d-3518-401a-92cf-6c75ff4c7526.png)
 Can you suggest anything to resolve this issue?

Reply:
hmm
try to reload TS server

Reply:
even on this screen, all parameters are required. we don't have any option params

Reply:
I have reloaded servers multiple times, nothing seems to help. And trying to solve this issue created a new question for me.
In validation, we pass the min, max values from this enum, so using {#limit} will take the values from this enum. Is there any advantage of using enum over {#limit} other than readability of code?

Reply:
I think it is because of cycle import. try to use relative import for `ValidationRulte` enum instead